pom.xml

```diff
@@ -9,10 +9,15 @@
         <version>2.17</version>
     </parent>
 
-    <name>Jenkins BlueOcean Pipeline Editor</name>
+    <name>Blue Ocean Pipeline Editor</name>
+    <description>
+        The Blue Ocean Pipeline Editor is the simplest way for anyone 
```

Comment:
dont want it to be `...way for anyone                        wanting to...` either.